libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/member/RoomMemberListFetcher.kt

```diff
@@ -111,10 +111,8 @@ internal class RoomMemberListFetcher(
                     val chunk = iterator.nextChunk(pageSize.toUInt())
                     val members = try {
                         // Load next chunk. If null (no more items), exit the loop
-                        chunk?.parallelMap(RoomMemberMapper::map) ?: break
+                        chunk?.parallelMap(ExportedRoomMemberMapper::map) ?: break
```

Comment:
From my previous tests, I think it was actually faster to drop the `parallel` usage here after changing to plain data classes, but I'd have to re-test it to be sure. Anyway, we can leave it as is and change it later if it's actually faster.